BREAKING.md

```diff
@@ -48,6 +49,12 @@ The following interfaces available in `@fluidframework/runtime-definitions` are
 -   `IGarbageCollectionSnapshotData`
 -   `IGarbageCollectionSummaryDetailsLegacy`
 
+### Op compression is enabled by default
+
+If the size of a batch is larger than 614kb, the ops will be compressed. After upgrading to this version, if batches exceed the size threshold, the runtime will produce a new type of op with the compression properties. To be able to open a document which contains this type of op, the runtime version needs to be at least `client_v2.0.0-internal.2.3.0`. Older clients will close with assert `0x3ce` and will not be able to open the documents until they upgrade.
```

Comment:
I'd be a bit more specific here. I.e. app developer has to ensure that the version of the app with this version of FF can be deployed to prod when app developer ensured that 99.99% of app sessions are running with FF version above X.